javascript_nodejs/14.nlp-with-dispatch/nlp-with-dispatch.bot

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
this sample uses LUIS and QnA, should the bot file include more services?

Reply:
By default, the .bot file does not have these service configurations. You need to create the LUIS, QnA Maker and dispatch services by following the instructions in readme.md. 

Alternative would be for us to invest in and deliver msbot configure services which would do this automatically for the user.

Reply:
We need to be consistent across the samples. Some have the list of all services, others do not.